### PR TITLE
Enable generation of inter-procedual control flow graph.

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Requires: https://github.com/rizinorg/rizin-testbins/pull/134

Adds the ability to generate a inter-procedual control flow graph.

One thing to discuss before merge:

The graph uses `RzGraphNodeInfo` instead of a simple `RzGraphNode`. This wastes space (at least the two `char *` members) because `iCFGs` will get enormous.

But if the graph is supposed to be returned by `rz_core_graph` it is basically forced to use it. Using only the node prevents printing it. Which would only be annoying for testing. 

**Test plan**

Test added

**Closing issues**

None

### CODE REVIEW

**Review by wargio — APPROVED** [review]

**Comment by wargio** [comment]
@Rot127 is missing the changes on the YAML file.

File: librz/util/graph.c [hunk 1]
```diff
@@ -174,7 +174,7 @@ RZ_API RzGraphNode *rz_graph_add_nodef(RzGraph *graph, void *data, RzListFree us
 
 /* remove the node from the graph and free the node */
 /* users of this function should be aware they can't access n anymore */
```
Comment (XVilka) [inline librz/util/graph.c]: Convert comment to doxygen while you are on it

**Review by XVilka — CHANGES_REQUESTED** [review]

**Review by XVilka — APPROVED** [review]

**Review by XVilka — CHANGES_REQUESTED** [review]
```
Missing type comment at <rizin/librz/core/cgraph.c:796:69> (token is not a comment)
Missing type comment at <rizin/librz/core/cgraph.c:817:72> (token is not a comment)
```
Also:
```
Run ./librz/core/cmd_descs/cmd_descs_generate.py --output-dir /tmp ./librz/core/cmd_descs/*.yaml
218d217
< static const RzCmdDescArg analysis_graph_icfg_args[2];
Error: Process completed with exit code 1.
```

**Comment by XVilka** [comment]
@Rot127 still not synchronized YAML and C/H command handlers files:
```
Run ./librz/core/cmd_descs/cmd_descs_generate.py --output-dir /tmp ./librz/core/cmd_descs/*.yaml
218d217
< static const RzCmdDescArg analysis_graph_icfg_args[2];
Error: Process completed with exit code 1.
```

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
Regarding this:

> The graph uses RzGraphNodeInfo instead of a simple RzGraphNode. This wastes space (at least the two char * members) because iCFGs will get enormous.
>
> But if the graph is supposed to be returned by rz_core_graph it is basically forced to use it. Using only the node prevents printing it. Which would only be annoying for testing.

Our graph API is in very poor shape, to be honest. We should have better graph data structures, just plain graphs and "enriched" with metainformation that is ready for printing: https://github.com/rizinorg/rizin/issues/992 (see also https://github.com/rizinorg/rizin/issues/397)

**Pull request merged**